arch/armv7m/reboot.c

```diff
@@ -17,11 +17,11 @@
 #include <sys/platform.h>
 
 #if defined(__CPU_STM32L4X6)
-#include <phoenix/arch/stm32l4.h>
+#include <phoenix/arch/armv7m/stm32/l4/stm32l4.h>
 #elif defined(__CPU_IMXRT117X)
-#include <phoenix/arch/imxrt1170.h>
+#include <phoenix/arch/armv7m/imxrt/11xx/imxrt1170.h>
 #else
```

Comment:
Thanks, for now I'll leave the ifdefs as they are because they're only needed in one file, but if ever necessary elsewhere I agree that this could be moved to separate header.